src/backend/distributed/sql/citus--9.5-1--10.0-1.sql

```diff
@@ -5,3 +5,22 @@
 #include "udfs/citus_finish_pg_upgrade/10.0-1.sql"
 
 #include "../../columnar/sql/columnar--9.5-1--10.0-1.sql"
+
+CREATE VIEW public.citus_tables AS
+SELECT
+  logicalrelid AS table_name,
+  CASE WHEN partkey IS NOT NULL THEN 'distributed' WHEN repmodel = 't' THEN 'reference' ELSE 'local' END AS citus_table_type,
+  column_to_column_name(logicalrelid, partkey) AS distribution_column,
+  colocationid AS colocation_id,
+  pg_size_pretty(citus_total_relation_size(logicalrelid)) AS table_size,
```

Comment:
I wonder if we should also have the byte version of this so that it is possible to check tables with a size greater than X from this view, would that be useful?

Reply:
This is mainly meant to be human-readable for novice users.

For querying based on size you can use `SELECT logicalrelid FROM pg_dist_partition WHERE citus_total_relation_size(logicalrelid) > 1000000000`